picker_wrapper.go

```diff
@@ -156,3 +195,84 @@ func (bp *pickerWrapper) close() {
 	bp.done = true
 	close(bp.blockingCh)
 }
+
+type stickyStoreEntry struct {
+	acw  *acBalancerWrapper
+	addr resolver.Address
+}
+
+type stickyStore struct {
+	mu    sync.Mutex
+	store map[string]*stickyStoreEntry
```

Comment:
Can this key not be a string? This key doesn't depend on service config right?
 
Also, How about instead of using a map we use 10 slices each with it's own mutex and convert the string into an int (without using something as expensive as hashing), and mod the int with 10 and use that slice for it?
One possible way of string to int could be:
```go
a := "some_string"
num := 0
for _, b := range []byte(a){
	num += int(b)
}
```